library/formatting.inc.php

```diff
@@ -216,3 +216,27 @@ function oeFormatAge($dobYMD, $nowYMD = '', $format = 0)
 
     return $age;
 }
+
+
+/**
+ * Returns the complete formatted stae string according the global date format
+ * @param $timestamp
+ * @return bool|string
+ */
+function oeTimestampFormatDateTime($timestamp)
+{
+    if(!$timestamp){
+        $timestamp = date('Y-m-d H:i');
```

Comment:
shouldn't this be:
``````
$timestamp = strtotime(date('Y-m-d H:i'));
``````
or maybe even this which seems simpler:
``````
$timestamp = strtotime("now");
``````